.github/workflows/frontend-lint.yml

```diff
@@ -54,3 +54,15 @@ jobs:
         cache-dependency-path: 'yarn.lock'
     - run: yarn install --immutable --check-cache
     - run: yarn run typecheck
+  lint-frontend-betterer:
+    name: Betterer
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v4
+      with:
+        node-version-file: '.nvmrc'
+        cache: 'yarn'
+        cache-dependency-path: 'yarn.lock'
+    - run: yarn install --immutable --check-cache
```

Comment:
Not really related to this PR but why are we doing cache-checks here? Have we switched to using Zero Installs?

>If the --check-cache option is set, Yarn will always refetch the packages and will ensure that their checksum matches what's 1/ described in the lockfile 2/ inside the existing cache files (if present). This is recommended as part of your CI workflow if you're both following the Zero-Installs model and accepting PRs from third-parties, as they'd otherwise have the ability to alter the checked-in packages before submitting them.

Reply:
yeah i don't know, i just copied from above 😅

Reply:
Pretty sure this is why every install takes ~2mins.

Reply:
It hought it seemed like a good idea at the time since we are accepting PRs from third-parties. I would even guess that removing it would have no effect since yarn detects it's running in github actions and automatically enables https://yarnpkg.com/features/security#hardened-mode